features/foutafhandeling.feature

```diff
@@ -0,0 +1,542 @@
+        | name        | code         | reason                      |
+        | bestaatniet | unknownParam | Parameter is niet verwacht. |
+
+Rule: de parameterwaarde moet aan de parameterspecificaties voldoen
+
+    Abstract Scenario: waarde van een queryparameter is niet correct
+        Als '<path><query string>' wordt aangeroepen
+        Dan bevat de response ten minste de volgende velden
+        | naam   | waarde                                        |
+        | title  | Een of meerdere parameters zijn niet correct. |
+        | status | 400                                           |
+        | code   | paramsValidation                              |
+        En bevat de response de volgende invalidParams
+        | name        | code   | reason                |
+        | <parameter> | <code> | <reason omschrijving> |
+
+        Voorbeelden:
+        | path                   | query string                                                              | parameter         | code             | reason omschrijving                                 |
+        | /adressen              | ?pandIdentificatie=0345100002016017&page=a                                | page              | integer          | Waarde is geen geldige integer.                     |
+        | /panden                | ?locatie=abc,def                                                          | locatie           | number           | Waarde is geen geldig decimaal getal.               |
+        | /adresseerbareobjecten | ?bbox=135207,457400,135418,457162&geconstateerd=Nee                       | geconstateerd     | boolean          | Waarde is geen geldige boolean.                     |
+        | /adresseerbareobjecten | ?bbox=135207,457400,135418,457162&oppervlakte[min]=-1&oppervlakte[max]=1  | oppervlakte[min]  | minimum          | Waarde is lager dan minimum 1.                      |
+        | /panden                | ?bbox=135207,457400,135418,457162&bouwjaar[min]=2000&bouwjaar[max]=20200] | bouwjaar[max]     | maximum          | Waarde is hoger dan maximum 9999.                   |
+        | /panden                | ?locatie=98095.02                                                         | locatie           | minItems         | Array bevat minder dan 2 items.                     |
+        | /panden                | ?locatie=98095.02,438495.09,0                                             | locatie           | maxItems         | Array bevat meer dan 2 items.                       |
+        | /adressen              | ?pandIdentificatie=0345100002016017X                                      | pandIdentificatie | pattern          | Waarde voldoet niet aan patroon ^[0-9]{16}$.        |
+        | /adresseerbareobjecten | ?bbox=135207,457400,135418,457162&type=fout                               | type              | enum             | Waarde heeft geen geldige waarde uit de enumeratie. |
+
+    Scenario: waarde van de padparameter is niet correct
```

Comment:
'Niet correct' is niet specifiek genoeg. Wat is er niet correct? Is het een type fout of een validatie fout?

Reply:
ik maak ervan "waarde van de padparameter is niet correct conform parameterspecificaties"